README.md

```diff
@@ -293,3 +279,60 @@ func startOrchextraContentManager() {
 	}
 }
 ```
+
+### Third-party providers
+
+OCM provides a way to configure some Third-party providers by sending some information to the SDK. All providers can be configured in:
+
+```swift
+OCM.shared.providers = ...
+```
+
+The list of supported providers is:
```

Comment:
I'd suggest to change it to this: "Currently supported providers are listed below:"